### PR TITLE
New Port: Toziuha Night: Dracula's Revenge

### PR DESCRIPTION
**Pull request by Ganimoth**
# New Port for Toziuha Night: Dracula's Revenge

## Game Information
- **Title**: Toziuha Night: Dracula's Revenge
- **URL**: https://dannygaray60.itch.io/toziuha-night-draculas-revenge

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/toziuha/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Toziuha Night - Dracula's Revenge.sh</path>
+    <name>Toziuha Night: Dracula's Revenge</name>
+    <desc>Toziuha Night Dracula's Revenge is a 2D side-scrolling action platformer with classicvania style. Travel through different linear maps set in a dark fantasy world; such as a gloomy forest and  Count Dracula's castle.
+Play as Xandria, a beautiful and skilled alchemist who, using an iron whip, fights against the most fearsome demons and other alchemists. Find out more about this tragic story of a girl who fights with a whip of iron chains, a cruel reminder of a tragic past.</desc>
+    <releasedate>20220321T000000</releasedate>
+    <developer> Danny Garay	</developer>
+    <publisher> Danny Garay	</publisher>
```
Comment (cdeletre) [inline ports/toziuha/gameinfo.xml:10]: The whitespaces can be removed I guess ?

```suggestion
    <developer>Danny Garay</developer>
    <publisher>Danny Garay</publisher>
```

File: ports/toziuha/Toziuha Night - Dracula's Revenge.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		$controlfolder/xdelta3 -d -s "gamedata/toziuha.pck" "patch/toziuhapatch.xdelta" "gamedata/toziuhapatched.pck" && rm "gamedata/toziuha.pck"
+		echo "Pck file from itch.io has been patched"
+    else
+	    echo "checksum does not match; wrong build/version of game"
+	fi
+	elif [ -f "gamedata/toziuha_night_dracula's_revenge.pck" ]; then
+	 # get toziuha_night_dracula's_revenge.pck checksum for Windows and Linux versions from Steam
+    checksum=$(md5sum "gamedata/toziuha_night_dracula's_revenge.pck" | awk '{ print $1 }')
+		# # Checksum for the Steam versio
+		if [ "$checksum" == "7a9a0252621e36cc053f1efec1a646ec" ]; then
```
Comment (cdeletre) [inline ports/toziuha/Toziuha Night - Dracula's Revenge.sh:49]: It's the same md5sum for itch and steam version. Is that alright ?

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/toziuha/Toziuha Night - Dracula's Revenge.sh (hunk 2 above)
Comment (Ganimoth) [reply ports/toziuha/Toziuha Night - Dracula's Revenge.sh:49]: It is, previously I had there different one for windows itch version.. until I realized the linux on itch and windows on steam are the same files, so the only one sticking out is the windows itch version. So I changed it to this so I dont need 2 separate patch files.

File: ports/toziuha/gameinfo.xml (hunk 1 above)
Comment (Ganimoth) [reply ports/toziuha/gameinfo.xml:10]: I guess they can

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**